### PR TITLE
build: add ARM64 to test matrix

### PR DESCRIPTION
**Pull request by glevand**
This finishes the work started by @crawford in https://github.com/coreos/ignition/pull/194.

This PR includes the gimme script submitted upstream https://github.com/travis-ci/gimme/pull/45.  If that is merged we can remove the gimme.local here.

### CODE REVIEW

File: build [hunk 1]
```diff
@@ -11,8 +11,15 @@ if [ ! -h gopath/src/${REPO_PATH} ]; then
 	ln -s ../../../.. gopath/src/${REPO_PATH} || exit 255
 fi
 
-export GOBIN=${PWD}/bin
+if [ -z "${GOARCH}" ] || [ "${GOARCH}" = "$(go env GOHOSTARCH)" ]; then
```
Comment (crawford) [inline build]: Eh, I don't like special cases. Just do `bin/${GOARCH}` unconditionally.

**Comment by crawford** [comment]
Woah, this is awesome. You are a beast.

File: build (hunk 1 above)
Comment (glevand) [inline build]: @crawford OK, for that I think we'll need something like ```export GOBIN="${PWD}/bin/$(go env GOARCH)" to handle when GOARCH is not set.

Comment (crawford) [inline build]: Arg, is it not always set?

Comment (marineam) [inline build]: Well, it will be after line 24...

**Comment by glevand** [comment]
Moved ```eval $(go env)`` up in build file.

**Comment by crawford** [comment]
Excellent. LGTM. Thank you so much.

**Pull request merged**